tools/org.eclipse.n4js.hlc/src/main/resources/org/eclipse/emf/ecore/plugin/plugin.properties

```diff
@@ -0,0 +1,220 @@
+#
+# EXPERMIMENTAL Pre-merged file, taken from following sources:
+#  - org.eclipse.emf.ecore_2.10.1.v20140901-1043.jar
+#
+#
+#
+#
+#
+#
+
+
+
+
+# /**
+#  * Copyright (c) 2002-2010 IBM Corporation and others.
```

Comment:
This is actually required by the headless compiler, other wise it will go off rails when running.
We can not commit it, but then we cannot build our our headless tool.